src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs

```diff
@@ -1020,10 +1035,49 @@ private void SetServicesFromManifest(AppManifest appManifest)
                     {
                         existingSvOption.IsSelected = true;
                     }
+
+                    var svcPresentInOptions = ServiceOptions.Exists(s => s.Name == svName);
+                    if (!svcPresentInOptions)
```

Comment:
What about case sensitivity?

Reply:
Great call -- are you allowed to use any case in CF manifests?

Reply:
This might take a couple quick rounds of testing... I don't know offhand how TAS will handle a mismatch between the manifest and what exists

Reply:
![image](https://user-images.githubusercontent.com/3947063/165833996-749fcb90-3752-43bb-a90d-8e35674fda08.png)

Reply:
🤷 sounds easier just to compare case-standardized values 😄 (commit incoming!)

Reply:
Sorry for the extra back and forth here, looks like you had it right originally... it is case-sensitive all the way around

Reply:
![image](https://user-images.githubusercontent.com/3947063/165834930-281e1191-57d5-4c79-b31d-570b0f9ed8ff.png)
The service binding that exists here is named `mStoreRegistry`

Reply:
Ah ok yeah I guess ~failing~ warning early is preferable then if we can catch case mismatches... (I was assuming with [`6d75b14`](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/326/commits/6d75b14abcc8f66598da5bdb68317c62deff4b1f) that it would be strictly better to ignore casing... nvm!)